config/config.toml

```diff
@@ -1,13 +1,24 @@
 [publisher]
-provider_engine = 'coin_gecko'
+
+# Set it to either 'coin_gecko' or 'pyth_replicator'. You need to provide
+# the configuration for the chosen engine as described below.
+provider_engine = 'pyth_replicator'
```

Comment:
minor nit: instead of hardcoding the provider types, it might be less brittle to have a list of each type of provider. So the config would look like:
- list of `pyth_replicator`'s providers
- list of `coin_gecko`'s providers

Reply:
I think it kind of defeats the purpose of having only 1 provider right? (In the past the code could have multiple provider but now it's only one single one)